doc/nrf/ug_matter_gs_testing_wifi_pc.rst

```diff
@@ -28,12 +28,40 @@ To use this setup, you need the following hardware:
 Configuring the environment
 ***************************
 
-To configure and use CHIP Tool for Linux or macOS, complete the following steps:
+To configure and use CHIP Tool for Linux or macOS with Matter over Wi-Fi, complete the following steps.
+
+.. rst-class:: numbered-step
+
+Program the sample
+==================
+
+Program the development kit for the Matter accessory device with one of available :ref:`matter_samples`.
+
+.. rst-class:: numbered-step
+
+Configure the CHIP Tool for Linux or macOS
+==========================================
+
+CHIP Tool for Linux or macOS is the default implementation of the :ref:`ug_matter_network_topologies_concepts_controller` role, recommended for the nRF Connect platform.
```

Comment:
Perhaps, this could be included from the previous docs.

Reply:
Done